pkg/server/status.go

```diff
@@ -38,18 +37,24 @@ func (c *criContainerdService) Status(ctx context.Context, r *runtime.StatusRequ
 		Type:   runtime.RuntimeReady,
 		Status: true,
 	}
-	// Use containerd grpc server healthcheck service to check its readiness.
-	resp, err := c.healthService.Check(ctx, &healthapi.HealthCheckRequest{})
-	if err != nil || resp.Status != healthapi.HealthCheckResponse_SERVING {
+
+	if c.client == nil {
```

Comment:
I did that only for this if some test cases we expect the error in that format. Happy to remove it . Dont have a strong opinion on this one.